recipes/examples/tv_trending.yml

```diff
@@ -20,6 +20,22 @@ source_libraries:
     folders:
       - '/path/to/Different Shows'
 
+# New playlist details
+new_playlist:
+  name: 'TV - Trending'
+  # Limit the age (in years) of items to be considered
+  # * 0 for no limit
+  max_age: 0
+  # Maximum number of items to keep in the library
+  max_count: 250
+  # Remove items that no longer exist in the source lists
+  remove_from_playlist: false
+  share_to_users:
+    - username1
```

Comment:
Any idea if this is possible with managed users?

Reply:
Yes, getting users from Plex returns managed/home users as well as Friends